src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationOddLineWrappingAndArrayInit.java

```diff
@@ -0,0 +1,30 @@
+/* Config:                                                                    //indent:0 exp:0
+ * This test-input is intended to be checked using following configuration:   //indent:1 exp:1
+ *                                                                            //indent:1 exp:1
+ * arrayInitIndent = 3                                                        //indent:1 exp:1
+ * basicOffset = 4                                                            //indent:1 exp:1
+ * braceAdjustment = 0                                                        //indent:1 exp:1
+ * caseIndent = 4                                                             //indent:1 exp:1
+ * forceStrictCondition = false                                               //indent:1 exp:1
+ * lineWrappingIndentation = 9                                                //indent:1 exp:1
+ * tabWidth = 4                                                               //indent:1 exp:1
+ * throwsIndent = 4                                                           //indent:1 exp:1
+ *                                                                            //indent:1 exp:1
+ *                                                                            //indent:1 exp:1
+ */                                                                           //indent:1 exp:1
+
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation; //indent:0 exp:0
+
+public class InputIndentationOddLineWrappingAndArrayInit { //indent:0 exp:0
+    interface MyInterface { //indent:4 exp:4
+        @interface SomeAnnotation { String[] values(); } //indent:8 exp:8
+        interface Info { //indent:8 exp:8
+            String A = "a"; //indent:12 exp:12
+        } //indent:8 exp:8
+        @MyInterface.SomeAnnotation(values = {       //indent:8 exp:8
+                MyInterface.Info.A, //indent:16 exp:11,17,53,54 warn
```

Comment:
`exp:11,17,53,54`
same here, indentation 53 or 54 sounds a bit weird

Reply:
53 is the indent level of first comment on line "24".
54 is the lineWrapping + lcurly pos.
Here is the logic:
```
        if (firstChildPos != NOT_EXIST) {
            expectedIndent = IndentLevel.addAcceptable(expectedIndent, firstChildPos, lcurlyPos
                    + getLineWrappingIndentation());
        }
```

Reply:
comment was unnecesarily placed too far, corrected.